hardware/opas/opas.py

```diff
@@ -335,8 +334,11 @@ def update(self):
         # TODO: 
         # update plot lines
         motor_name = self.plot_motor.read()
-        motor_position = self.driver.motor_positions[motor_name].read()
-        self.plot_h_line.setValue(motor_position)
+        try:
+            motor_position = self.driver.motor_positions[motor_name].read()
+            self.plot_h_line.setValue(motor_position)
+        except:
+            pass
```

Comment:
Honestly, this change may be able to be reverted, I think it may have been fixed by moving the code from `initialize` to `__init__`